src/TMU.cpp

```diff
@@ -13,9 +13,17 @@ uint16_t TMU::getObjectDictionarySize() const {
     return OBJECT_DICTIONARY_SIZE;
 }
 
-void TMU::updateTemps() {
+void TMU::process() {
+    uint16_t temp_array[4] = {0};
```

Comment:
Using this array instead of `thermTemps` prevents the data from being read by CANopen properly. You should go back to using that array